pkg/etc/wavefront/wavefront-proxy/wavefront.conf.default

```diff
@@ -20,6 +20,26 @@ server=CHANGE_ME
 # 3. Click *API access*.
 token=CHANGE_ME
 
+# To add a proxy, you need to use an existing API token with AOA service proxy role. If you have no API token yet, you
+# can create one under your account page in VMWare Cloud Service.
+cspAPIToken=CSP_API_TOKEN_HERE
+
+# To add a proxy, you need to use an existing App ID, App Secret for server to serve type of app with AOA service proxy role.
+# If you have no App ID and App Secret yet, you can create one for server to serve type of app under Organization/OAuth
+# Apps menu item in VMWare Cloud Service. Note: Proxy, based on OAuth apps, has no expiration time.
+cspAppId=CSP_APP_ID_HERE
+
+# To add a proxy, you need to use an existing App ID, App Secret for server to serve type of app with AOA service proxy role.
+# If you have no App ID and App Secret yet, you can create one for server to serve type of app under Organization/OAuth
+# Apps menu item in VMWare Cloud Service. Note: Proxy, based on OAuth apps, has no expiration time.
+cspAppSecret=CSP_APP_SECRET_HERE
+
+# The CSP organisation ID.
+cspOrgId=CSP_ORG_ID_HERE
+
+# CSP console URL. This will be used in many places like getting token. If you wish to use the csp stg url, comment the line.
+cspBaseUrl=https://console.cloud.vmware.com
```

Comment:
this doesn't match the default value, and commented

Reply:
> this doesn't match the default value, and commented

fixed